### PR TITLE
Update lambda-auto-instrument.md

### PR DESCRIPTION
**Pull request by tylerbenson**
Instructions were missing the step to configure `AWS_LAMBDA_EXEC_WRAPPER`.  Also fixed the java supported versions.

cc @cartersocha 

---

**Preview**: https://deploy-preview-2945--opentelemetry.netlify.app/docs/faas/lambda-auto-instrument/

### CODE REVIEW

File: content/en/docs/faas/lambda-auto-instrument.md [hunk 1]
```diff
@@ -83,6 +81,12 @@ and the package on [PyPi](https://pypi.org/project/opentelemetry-api/).
 {{% /tab %}}
 {{< /tabpane >}}
 
+### Configure `AWS_LAMBDA_EXEC_WRAPPER`
+
+Change the entry point of your application by setting `AWS_LAMBDA_EXEC_WRAPPER=/opt/otel-handler` for Node.js or java and
+`AWS_LAMBDA_EXEC_WRAPPER=/opt/otel-instrument` for python.  These wrapper scripts will invoke your lambda application
```
Comment (chalin) [inline content/en/docs/faas/lambda-auto-instrument.md:87]:
```suggestion
Change the entry point of your application by setting `AWS_LAMBDA_EXEC_WRAPPER=/opt/otel-handler` for Node.js or Java, and
`AWS_LAMBDA_EXEC_WRAPPER=/opt/otel-instrument` for Python.  These wrapper scripts will invoke your Lambda application
```

File: content/en/docs/faas/lambda-auto-instrument.md [hunk 2]
```diff
@@ -56,7 +55,6 @@ For example, to only enable auto-instrumentation for Lambda and the AWS SDK, you
 would have to set the following environment variables:
 
     ```bash
```
Comment (chalin) [inline content/en/docs/faas/lambda-auto-instrument.md:57]: Unindent this code block.

Comment (chalin) [reply content/en/docs/faas/lambda-auto-instrument.md:57]: Also the list items at lines 51 and 52 above need to be unindented. 

Would you mind fixing them? Thanks!

**Review by chalin — CHANGES_REQUESTED** [review]
Some of the text is misformatted (see inline comments for fixes) as can be seen from this screenshot:

> image

File: content/en/docs/faas/lambda-auto-instrument.md (hunk 2 above)
Comment (chalin) [reply content/en/docs/faas/lambda-auto-instrument.md]: Thanks!

**Review by chalin — APPROVED** [review]
LGTM

Will let others validate content.

**Review by svrnm — APPROVED** [review]

**Pull request merged**